### PR TITLE
discover mobile test

### PR DESCRIPTION
**Pull request by liuliu-dev**

### CODE REVIEW

**Comment by liuliu-dev** [comment]
the changes of the discover page haven't gone prod yet, cypress tests will fail because of that.

File: cypress/integration/productionTests/publicPaths/render/discover/index.spec.ts [hunk 1]
```diff
@@ -37,13 +36,11 @@ describe(`${pageName} renders expected components on different devices`, () => {
...
   ];
 
-  const iPhoneTests = testMobileRepoList("[data-cy=discover-repo-lists]");
-
   const devices = allDevicesDiffTestsForSignedOut(
     pageName,
-    desktopTests,
-    desktopTests,
-    iPhoneTests,
+    tests,
```
Comment (tbantle22) [inline cypress/integration/productionTests/publicPaths/render/discover/index.spec.ts:41]: Can use `allDevicesForSignedOut` for this so you only have to pass in `tests` twice

File: cypress/integration/productionTests/publicPaths/render/discover/withQuery.spec.ts [hunk 2]
```diff
@@ -51,13 +48,11 @@ describe(`${pageName} renders expected components on different devices`, () => {
...
   ];
 
-  const iPhoneTests = testMobileRepoList("[data-cy=discover-repo-lists]");
-
   const devices = allDevicesDiffTestsForSignedOut(
     pageName,
-    desktopTests,
-    desktopTests,
-    iPhoneTests,
+    tests,
```
Comment (tbantle22) [inline cypress/integration/productionTests/publicPaths/render/discover/withQuery.spec.ts:53]: Same for these

**Review by tbantle22 — APPROVED** [review]
LGTM! Can merge when changes on prod and the tests in this PR pass

**Pull request merged**